### PR TITLE
Update modify container images document

### PR DESCRIPTION
**Pull request by puneetse**
This PR consolidate and update examples and cleans up RST syntax warnings along the way.

Previously 2 multi-stage build examples exists: redis (2 layers) and cgit (3 layers). This was confusing because the cgit example was unrelated to the redis example, but used steps done in it to complete successfully. The redis example has been removed in favor of one reference example.

### CODE REVIEW

File: source/guides/maintenance/container-image-modify.rst [hunk 1]
```diff
@@ -131,21 +133,27 @@ OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build c
...
+   OS version. In this example, four base layers must be rebuilt.
 
-#. Change directory into :file:`dockerfiles`.
 
-First layer
------------
+First layer: os-core
+--------------------
+
+#. Change directory into :file:`dockerfiles`.
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Ln 166:  When I run this command, output shows `update successful`; however, an error message appears just like above: 
`swupd: error while loading shared libraries: libsystemd.so.0: cannot open shared object file: No such file or directory`

File: source/guides/maintenance/container-image-modify.rst [hunk 2]
```diff
@@ -162,62 +170,25 @@ First layer
...
-#. Change the :file:`httpd/Dockerfile` to use the desired OS
-   version; in this case, the build version is 31090.
+#. Change the :file:`httpd/Dockerfile` to use the version-specific
+   *os-core:31090* image that was previously built.
 
 #. Run :command:`git diff`.
 
-   The output shows a diff of a modified :file:`clearlinux/httpd` Dockerfile that uses the previously built clearlinux/os-core:31090.
+   The output shows a diff of a modified :file:`clearlinux/httpd` Dockerfile
+   that uses the previously built clearlinux/os-core:31090.
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Before Ln 215, add: 
```
.. code-block:: bash
   cd ..
```

File: source/guides/maintenance/container-image-modify.rst [hunk 3]
```diff
@@ -252,8 +223,8 @@ The next layer is :file:`clearlinux/httpd`.
       --build-arg https_proxy=$https_proxy \
       -t clearlinux/httpd:31090 httpd/
 
-Fourth layer
-------------
+Third layer: cgit
+-----------------
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Before Ln 265, add: 
```
.. code-block:: bash
   cd ..
```

File: source/guides/maintenance/container-image-modify.rst [hunk 4]
```diff
@@ -29,11 +30,9 @@ Prerequisites
...
 
-Examples
-********
 
 Example 1: Add a bundle
-=======================
+***********************
 
 In this example, we add :command:`wget` to the **clearlinux/redis**
 Dockerfile.
```
[the remark this answers is not in the document]
Comment (puneetse) [reply source/guides/maintenance/container-image-modify.rst]: There should not be sudo here. This is not an installation of wget to the host system. The container runs as root.

File: source/guides/maintenance/container-image-modify.rst [hunk 5]
```diff
@@ -51,7 +50,8 @@ Dockerfile.
 
 #. Run :command:`git diff`.
 
-   The output shows the edits made after adding :command:`wget` in the clearlinux/redis Dockerfile.
+   The output shows the edits made after adding :command:`wget` in the
+   clearlinux/redis Dockerfile.
```
[the remark this answers is not in the document]
Comment (puneetse) [reply source/guides/maintenance/container-image-modify.rst]: This indicates another issue with your environment. No libsystemd tells me you're likely in a console inside a container when you shouldn't be,

File: source/guides/maintenance/container-image-modify.rst (hunk 1 above)
Comment (puneetse) [reply source/guides/maintenance/container-image-modify.rst]: This indicates another issue with your environment. No libsystemd tells me you're likely in a console inside a container when you shouldn't be.

File: source/guides/maintenance/container-image-modify.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/guides/maintenance/container-image-modify.rst]: Fair. My environment was different.

File: source/guides/maintenance/container-image-modify.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/guides/maintenance/container-image-modify.rst]: I defer to your environment, like I said. But no, I wasn't working inside a container.

File: source/guides/maintenance/container-image-modify.rst [hunk 6]
```diff
@@ -29,29 +30,36 @@ Prerequisites
...
+   the software. The output should tell you that :command:`wget` is available
+   in the *wget* bundle.
 
-#. Open a an editor to modify the Dockerfile with
-   :command:`$EDITOR Dockerfile`.
+#. Open a an editor to modify the Dockerfile.
+
+   .. code-block:: bash   
+
+      $EDITOR redis/Dockerfile
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Cool!

File: source/guides/maintenance/container-image-modify.rst [hunk 7]
```diff
@@ -93,131 +102,103 @@ Dockerfile.
...
-OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build command line.
-
-#. Open a Terminal on |CL| and enter the
-   :file:`dockerfiles/machine-learning-ui` directory.
+OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build
+command line. 
 
 #. Rebuild the :file:`clearlinux/machine-learning-ui`. Add an extra build
-   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build version is 31090.
+   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Example 2, Step 1: 
Is it possible you could follow the same pattern in this example as you did in Example 1, Step 2-3? (Ln 50). 
I suppose there are some who know what you mean when you write, "Add and extra build argument :command:`swupd_args="-m 31110"`

I like how explicit the steps are in Example 1.

File: source/guides/maintenance/container-image-modify.rst [hunk 8]
```diff
@@ -93,131 +102,103 @@ Dockerfile.
...
 
-First layer
------------
+First layer: os-core
+--------------------
 
 #. Rebuild the first layer, *os-core*. Add an extra build argument
    :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build
-   version is 31090.
+   version is 31110.
```
Comment (mvincerx) [inline source/guides/maintenance/container-image-modify.rst]: Example 3, First layer, Step 1: 
Is it possible you could follow the same pattern in this example as you did in Example 1, Step 2-3? (Ln 50).

**Comment by mvincerx** [comment]
LGTM

File: source/guides/maintenance/container-image-modify.rst (hunk 8 above)
Comment (mvincerx) [reply source/guides/maintenance/container-image-modify.rst]: Thanks for clarifying, @puneetse. Confusion caused by a lack of GH to render code-blocks with emphasize lines.

**Pull request merged**